packages/react-components/react-infolabel-preview/stories/InfoLabel/InfoLabelPatternDecision.md

```diff
@@ -1,38 +0,0 @@
-<details>
```

Comment:
Should we be removing this file altogether?

Reply:
It's not removed, it was renamed, but appears as removed from react-infolabel-preview and as renames in react-infolabel